lib/monotime.h

```diff
@@ -132,6 +132,37 @@ static inline const char *frrtime_to_interval(time_t t, char *buf,
 	return buf;
 }
 
+static inline ssize_t frr_realtime_to_string(struct timeval *real, char *buf,
+					     size_t sz)
+{
+	ssize_t written = -1;
+	struct tm *lm;
+
+	lm = localtime((const time_t *)&real->tv_sec);
+
+	if (lm) {
+		written = (ssize_t)strftime(buf, sz, "%Y-%m-%d %H:%M:%S", lm);
+		if ((written > 0) && ((size_t)written < sz)) {
+			int w;
+
+			w = snprintf(buf + written, sz - (size_t)written,
+				     ".%06d", (int)real->tv_usec);
```

Comment:
This is kind of non obvious, it ends up looking like a decimal on the current wall time and not a totally unrelated value as it really is.